source/mainnet/net/web3-id/wallet.rst

```diff
@@ -0,0 +1,139 @@
+.. _web3id-wallet:
+.. include:: ../../variables.rst
+
+.. meta::
+    :description lang=en:
+        Web3 ID allows you to add and use verifiable credentials in the wallet for interactions with services that require verification.
+
+=====================
+Web3 ID in the |bw|
+=====================
+
+Web3 :ref:`verifiable credentials<glossary-verifiable-credential>` are available in the |bw|. They might have any of the following states:
+
+- Pending: the verifiable credential is in the process of being issued
+- Active: the verifiable credential is issued and valid
+- Revoked: the verifiable credential is no longer valid because the holder or issuer has canceled it
+- Not activated: the verifiable credential is issued but is not yet active because its *valid from date* has not passed, e.g. a concert ticket
+- Expired: the verifiable credential is past the *valid until date*, e.g. a concert ticket
+
+Add verifiable credentials to the wallet
+========================================
+
+When choosing to add verifiable credentials from an issuer, make sure that you *trust* the issuer.
```

Comment:
I am not sure I would stress the trust here. I am not sure what the intent is.

The trust is really on the verifier only. There is no harm in having credentials from bogus issuers in the wallet.

Is the intent here that since in order to get a credential, you usually have to reveal some information to the issuer, that you then need to trust the issuer to not misuses that information?

Reply:
The last part.